test/integ-new/main_new_test.go

```diff
@@ -220,6 +220,63 @@ func TestUIServiceHealthCheck(t *testing.T) {
 	}
 }
 
+func TestCertRotatorService(t *testing.T) {
+	t.Parallel()
+
+	cases := []struct {
+		name           string
+		path           string
+		wantStatusCode int
+	}{
+		{
+			name:           "cert_rotator_health_check",
+			path:           "/health",
+			wantStatusCode: http.StatusOK,
+		},
+		{
+			name:           "cert_rotator_key_rotation",
```

Comment:
It doesn't seem we verify the key was actually rotated? Shouldn't we do that?

Reply:
IIRC, the key won't actually rotate since the current version not expired yet. We can confirm the primary key is still the first version if you think it is necessary.

Reply:
+1

Reply:
So are you saying we're going to lose the test coverage in https://github.com/abcxyz/jvs/blob/main/test/integ/main_test.go#L326?

Reply:
right, should these be covered in unit test instead?

Reply:
How to cover them in unit test? We won't connect to KMS in unit test

Reply:
True, then we need to add some more test cases.

Reply:
Create issue https://github.com/abcxyz/jvs/issues/360.  Will do this as a follow up.

Reply:
Discussed offline. Will follow up in a separate PR